app/assets/javascripts/discourse/components/tag-list.js.es6

```diff
@@ -19,5 +19,16 @@ export default Ember.Component.extend({
   @computed("category.fullSlug")
   categoryClass(slug) {
     return slug && `tag-list-${slug}`;
+  },
+
+  @computed("tagGroupName")
+  tagGroupNameClass(groupName) {
+    if (groupName) {
+      var groupName = groupName
```

Comment:
```suggestion
      const groupName = groupName
```

Reply:
Here groupName being in the function params is considered already declared as mutable and doesn't needs a new declaration